apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -1906,160 +1923,168 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
             }
 
             { /* Pushed Authorization Requests*/ }
-            <Grid.Row columns={ 2 }>
-                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
-                    <Divider />
-                    <Divider hidden />
-                </Grid.Column>
-                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
-                    <Heading as="h4">
-                        { t("console:develop.features.applications.forms.inboundOIDC.sections" +
-                            ".pushedAuthorization.heading") }
-                    </Heading>
-                    <Field
-                        ref={ requirePushedAuthorizationRequests }
-                        name={ "requirePushAuthorizationRequest" }
-                        label=""
-                        required={ false }
-                        type="checkbox"
-                        value={ initialValues?.pushAuthorizationRequest?.requirePushAuthorizationRequest
-                            ? [ "requirePushAuthorizationRequest" ]
-                            : [] }
-                        children={ [
-                            {
-                                label: t("console:develop.features.applications.forms.inboundOIDC.sections" +
-                                    ".pushedAuthorization.fields.requirePushAuthorizationRequest.label"),
-                                value: "requirePushAuthorizationRequest"
-                            }
-                        ] }
-                        readOnly={ readOnly }
-                    />
-                    <Hint>
-                        { t("console:develop.features.applications.forms.inboundOIDC.sections" +
-                            ".pushedAuthorization.fields.requirePushAuthorizationRequest.hint") }
-                    </Hint>
-                </Grid.Column>
-            </Grid.Row>
+            { ApplicationTemplateNames.STANDARD_BASED_APPLICATION === template?.name && (
+                <Grid.Row columns={ 2 }>
+                    <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                        <Divider />
+                        <Divider hidden />
+                    </Grid.Column>
+                    <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
+                        <Heading as="h4">
+                            { t("console:develop.features.applications.forms.inboundOIDC.sections" +
+                                ".pushedAuthorization.heading") }
+                        </Heading>
+                        <Field
+                            ref={ requirePushedAuthorizationRequests }
+                            name={ "requirePushAuthorizationRequest" }
+                            label=""
```

Comment:
Shall we remove this empty prop?

Reply:
removed with [db451e6](https://github.com/wso2/identity-apps/pull/4616/commits/db451e67d57f674883d760ba9b0831bb14accd8f)